File: HEADBOB.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
	int t;
	cin >> t;
	while(t--)
	{
		int n,fy=0,fi=0,fn=0;
		cin >> n;
		while(n--)
		{
			char a;
			cin >> a;
			if( a == 'Y')
				fy=1;
			if( a == 'I')
			 	fi=1;
			if( a == 'N')
				fn=1;
		}

		if(fi == 1)
			cout << "INDIAN\n";
		else if(fy == 1)
			cout << "NOT INDIAN\n";
		else
			cout << "NOT SURE\n";
	}
	return 0;
}

File: chef/HEADBOB.py
t = input()
while(t > 0):
	t -= 1
	n = input()
	n=0;y=0;ind=0
	s = raw_input()
	for i in range(0,len(s)):
		if( s[i] == 'N' ):
			n += 1
		elif( s[i] == 'Y' ):
			y += 1
		elif( s[i] == 'I' ):
			ind += 1

	if(ind != 0):
		print 'INDIAN'
	elif(y != 0):
		print 'NOT INDIAN'
	else:
		print 'NOT SURE'